### PR TITLE
Feat: added cards

### PR DESCRIPTION
**Pull request by Prabhav-Sharma**
Added 6 types of Cards
1. Cards with badges
2. Cards with dismiss
3. Cards with text overlay
4. Text only cards
5. Vertical/Horizontal cards
6. Cards with shadow

### CODE REVIEW

File: Documentation/Components/Card/card.html [hunk 1]
```diff
@@ -0,0 +1,162 @@
...
+                    <button class="action-btn"><img class="card-icon" src="/Documentation/Components/images/bookmark.svg"></button>
+                </div>
+            </div>
+        </div>
+    </div>
+    
+    <div>
+        <h5>Card with Shadow</h5>
+        <div class="card flex-column card-shadow">
+            <img class="card-img" src="https://picsum.photos/300/150">
```
Comment (ashishnnnnn) [inline Documentation/Components/Card/card.html]: You could give here "alt" attribute to image

**Comment by Mohitkumar33** [comment]
Your cards may look more good if you reduce border radius

**Review by ashishnnnnn — CHANGES_REQUESTED** [review]

**Review by deepak29-git — COMMENTED** [review]
Everything looking good and clean code

File: Documentation/Components/Card/card.html (hunk 1 above)
Comment (Prabhav-Sharma) [reply Documentation/Components/Card/card.html]: Done.

**Comment by Prabhav-Sharma** [comment]
> Your cards may look more good if you reduce border radius

I've reduced the border-radius, they don't look good?

**Pull request merged**